src/tools/ci/serve.py

```diff
@@ -0,0 +1,25 @@
+# Server for the Verificarlo CI report. This is simply a wrapper to avoid
+# calling Bokeh directly.
+
+import os
+
+
+# Entry point of vfc_ci serve
+def run(directory, show, port, allow_origin, logo_url):
+
+    # Prepare arguments
+    directory = "directory %s" % directory
+
+    show = "--show" if show else ""
+
+    logo = ""
+    if logo_url is not None:
+        logo = "logo %s" % logo_url
+
+    dirname = os.path.dirname(__file__)
+
+    # Call the "bokeh serve" command on the system
+    command = "bokeh serve %s/vfc_ci_report %s --allow-websocket-origin=%s:%s --port %s --args %s %s" \
```

Comment:
is there a way to use python call instead of system?

Reply:
This seems possible, but might not be obvious because the report serves some static resource, uses a Jinja template, etc... which might be tricky to set up manually. This could be implemented in a future commit if we want to merge quickly.

Reply:
ok, can you add a comment to remember it? And resolve this conversation so that we can proceed on the merge operation?